sandbox/trim-low-abund.py

```diff
@@ -40,17 +40,17 @@ def output_single(read, trim_at=None):
     name = read.name
     sequence = read.sequence
 
-    accuracy = None
-    if hasattr(read, 'accuracy'):
-        accuracy = read.accuracy
+    quality = None
+    if hasattr(read, 'quality'):
+        quality = read.quality
 
     if trim_at is not None:
         sequence = sequence[:trim_at]
-        if accuracy:
-            accuracy = accuracy[:trim_at]
+        if quality:
+            quality = quality[:trim_at]
 
-    if accuracy:
-        return "@%s\n%s\n+\n%s\n" % (name, sequence, accuracy)
+    if quality:
```

Comment:
Note, this has already been changed to use write_record in another PR (#759) that is waiting for a code review...